libcloud/common/openstack_identity.py

```diff
@@ -940,32 +942,32 @@ def __init__(self, auth_url, user_id, key, tenant_name=None,
                             domain to scope the token to.
         :type domain_name: ``str``
 
-        :param token_scope: Whether to scope a token to a "project" or a
-                         "domain"
+        :param token_scope: Whether to scope a token to a "project", a
+                            "domain" or "unscoped"
         :type token_scope: ``str``
         """
         super(OpenStackIdentity_3_0_Connection,
               self).__init__(auth_url=auth_url,
                              user_id=user_id,
                              key=key,
                              tenant_name=tenant_name,
+                             domain_name=domain_name,
+                             token_scope=token_scope,
                              timeout=timeout,
                              parent_conn=parent_conn)
```

Comment:
I would suggest to change that code to:
```python
def __init__(self, *args, **kwargs):
    # Docstring
    super(OpenStackIdentityConnection, self).__init__(*args, **kwargs)
```
So we don't have to specify the signature twice.
But I know that `kwargs` are not appreciated, so tell me if you want me to do the change or not.